File: app/src/main/java/cn/imgrocket/imgrocket/UserActivity.java
package cn.imgrocket.imgrocket;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.view.MotionEvent;
import android.view.View;
import android.widget.RelativeLayout;

public class UserActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_user);
        RelativeLayout layoutItem1 = findViewById(R.id.user_layout_item1);
        RelativeLayout layoutItem2 = findViewById(R.id.user_layout_item2);
        RelativeLayout layoutItem3 = findViewById(R.id.user_layout_item3);
        layoutItem1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
        layoutItem2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
        layoutItem3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
    }
}

File: app/src/main/java/cn/imgrocket/imgrocket/UserActivity.kt
package cn.imgrocket.imgrocket

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_user.*

class UserActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_user)
        user_layout_item1.setOnClickListener {
            Toast.makeText(this,"hello",Toast.LENGTH_LONG).show();
        }
        user_layout_item2.setOnClickListener { }
        user_layout_item3.setOnClickListener { }
    }
}